api/applications/applicationRouter.js

```diff
@@ -1,16 +1,13 @@
 const express = require('express');
 // const authRequired = require('../middleware/authRequired');
-const Application = require('./applicationModel');
 const router = express.Router();
 // const { adminRequired } = require('../middleware/permissionsRequired.js');
 const {
-  cacheSignUpData,
-  sendData,
   checkApplicationExists,
   checkRole,
 } = require('../middleware/applicationMiddleware');
+const { v4: uuidv4 } = require('uuid');
```

Comment:
good use of the Universally unique identifier library as this isn't auth-related, just used to generate profile identifiers.